js/RecentDonations.js

```diff
@@ -0,0 +1,196 @@
+(function() {
+var app = angular.module('RecentDonations', []);
+app.controller('RecentDonationsControl', function($scope) {
+
+    var urlVars = GLOBALS.getUrlVars();
+    var count = Number(urlVars['count']);
```

Comment:
I _think_ you could do this like `var count = Number(urlVars['count']) || 5;` if you wanted to wrap it into 1 line.